docs/configuring-playbook-bot-mjolnir.md

```diff
@@ -28,7 +28,7 @@ If you use curl, you can get an access token like this:
 
 ```
 curl -X POST --header 'Content-Type: application/json' -d '{
-    "identifier": { "type": "m.id.user", "user": "bot.mjolnir" },
+    "identifier": { "type": "m.id.user", "user": "bot.mjolnir"},
```

Comment:
Same here - leading space in the `{` block, without trailing space before `}`.

Perhaps removing the leading space will also make yamllint happy?

Reply:
That could be my mistake. I did not want to edit all these changes by myself, so in the end i did most of these changes via find and replace.